kafka/consume.go

```diff
@@ -176,8 +245,8 @@ func (cc *ConfluentConsumer) setupRebalanceHandler(offset *int64) {
 				// if we have an initial offset we need to set it
 				if cc.conf.Consumer.InitialOffset != nil {
 					once.Do(func() {
-						log.WithField("kafka_offset", *offset).Debug("Skipping Kafka assignment given by coordinator after rebalance in favor of resetting the offset")
-						partitions = kafkalib.TopicPartitions{{Topic: &cc.conf.Topic, Offset: kafkalib.Offset(*offset)}}
+						log.WithField("kafka_offset", *cc.conf.Consumer.InitialOffset).Debug("Skipping Kafka assignment given by coordinator after rebalance in favor of resetting the offset")
```

Comment:
Took the opportunity to clean up the rebalance handler by using the InitialOffset from the config directly.